tdrs-frontend/src/components/PrivateTemplate/PrivateTemplate.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+
+
+export default function PrivateTemplate({ title, children }) {
```

Comment:
One of the QASP items refers to documentation, should we add documentation for this component? `PrivateTemplate` alone doesn't tell me what this component is doing or how to use it.